### PR TITLE
Create mqueue file system

### PR DESCRIPTION
**Pull request by Rachid-Koucha**
Added mqueue file system to be able to experiment POSIX mqueues isolation in IPC namespaces

Signed-off-by: Rachid Koucha <rachid.koucha@gmail.com>

### CODE REVIEW

**Comment by brauner** [comment]
jenkins: test this please

**Comment by brauner** [comment]
This breaks all tests. :/

**Comment by stgraber** [comment]
https://github.com/lxc/lxc/pull/3299 should fix this

File: templates/lxc-busybox.in [hunk 1]
```diff
@@ -237,6 +238,7 @@ lxc.cap.drop = sys_module mac_admin mac_override sys_time
 
 lxc.mount.auto = cgroup:mixed proc:mixed sys:mixed
 lxc.mount.entry = shm dev/shm tmpfs defaults 0 0
+lxc.mount.entry = mqueue dev/mqueue mqueue defaults 0 0
```
Comment (stgraber) [inline templates/lxc-busybox.in]: This should use `defaults,create=dir`

**Comment by brauner** [comment]
This needs a rebase now.

File: templates/lxc-busybox.in (hunk 1 above)
Comment (Rachid-Koucha) [reply templates/lxc-busybox.in]: So, we can remove mkdir of /dev/shm in :

  fstree="\
${rootfs}/selinux \
${rootfs}/dev \
${rootfs}/home \
${rootfs}/root \
${rootfs}/etc \
${rootfs}/etc/init.d \
${rootfs}/bin \
${rootfs}/usr/bin \
${rootfs}/sbin \
${rootfs}/usr/sbin \
${rootfs}/proc \
${rootfs}/sys \
${rootfs}/mnt \
${rootfs}/tmp \
${rootfs}/var/log \
${rootfs}/var/run \
${rootfs}/dev/pts \
${rootfs}/dev/shm \  <-------------------------------- Useless ?
${rootfs}/lib \
${rootfs}/usr/lib \
${rootfs}/lib64 \
${rootfs}/usr/lib64"

**Pull request closed without merging**